src/QMCDrivers/QMCDriverFactory.cpp

```diff
@@ -261,9 +330,33 @@ std::unique_ptr<QMCDriverInterface> QMCDriverFactory::createQMCDriver(xmlNodePtr
     APP_ABORT("QMCDriverFactory::createQMCDriver : method=\"linear_batch\" is not safe with CPU mixed precision. "
               "Please use full precision build instead.");
 #endif
-    auto opt = QMCWFOptLinearFactoryNew(cur, project_data_, emi, qmc_system,
-                                        MCPopulation(comm->size(), comm->rank(), &qmc_system, primaryPsi, primaryH),
-                                        qmc_system.getSampleStack(), comm);
+    app_summary() << "\n========================================"
+                     "\n  Reading WFOpt driver XML input section"
+                     "\n========================================"
+                  << std::endl;
+
+    QMCDriverInput qmcdriver_input;
+    VMCDriverInput vmcdriver_input;
+    try
+    {
+      qmcdriver_input.readXML(cur);
+      vmcdriver_input.readXML(cur);
+    }
+    catch (const std::exception& e)
+    {
+      throw UniformCommunicateError(e.what());
+    }
+
+    if (qmcdriver_input.get_estimator_manager_input())
+      throw UniformCommunicateError("Local Estimator sections are not allowed for Optimizer drivers!");
```

Comment:
This error message doesn't reflect the actual implementation. Could you print out a warning saying
```
The batched wavefunction optimization driver ignores local estimator input.
```

At the moment, please make it user-friendly as a warning instead of a stopper.